pttrack/models.py

```diff
@@ -232,6 +233,10 @@ class Patient(Person):
     preferred_contact_method = models.ForeignKey(ContactMethod, blank=True,
                                                  null=True)
 
+    # If the patient is in clinic and needs a workup, that is specified by needs_workup. Default value is false for all the previous patients
+
+    needs_workup = models.BooleanField(default=False)
```

Comment:
I did this for migration purposes because I did not want every patient to become True since that is the default value. I can do the migration on halstead, and then change this to True so that future patients will be handled correctly.